learning_assistant/api.py

```diff
@@ -103,7 +103,7 @@ def get_block_content(request, user_id, course_id, unit_usage_key):
     return cache_data['content_length'], cache_data['content_items']
 
 
-def render_prompt_template(request, user_id, course_run_id, unit_usage_key, course_id):
+def render_prompt_template(request, user_id, course_run_id, unit_usage_key, course_id, template_string):
     """
     Return a rendered prompt template, specified by the LEARNING_ASSISTANT_PROMPT_TEMPLATE setting.
```

Comment:
If `LEARNING_ASSISTANT_PROMPT_TEMPLATE` is remove, we might want to update this description.

Reply:
good catch 👍